controllers/parameters.go

```diff
@@ -34,14 +34,15 @@ const (
 )
 
 // filterPrefixedParameters removes all the reserved keys from the
-// replicationclass which are matching the prefix.
+// replication class which are matching the prefix.
```

Comment:
replicationclass single word please revert it back

Reply:
@Madhu-1 reverted this particular change. ptal.